src/main/java/org/openRealmOfStars/player/SpaceRace/SpaceRaceUtility.java

```diff
@@ -160,4 +156,12 @@ public static String getRandomName(final SpaceRace race,
     return sb.toString();
   }
 
+  private static void addSecondaryName(final SpaceRace race, StringBuilder sb, String secondaryName) {
```

Comment:
Good idea to add method for this. Javadocs are missing. Also parameters should have final keyword and then return name as a String. Running command mvn verify should run checkstyle checker which should have complain about those.

Reply:
Hi @tuomount I have updated the PR, please help me review it. Thanks